src/window.c

```diff
@@ -572,6 +603,9 @@ do_window(
 		// FALLTHROUGH
     case ']':
     case Ctrl_RSB:
+		if (!check_can_set_curbuf_disabled())
+		    break;
```

Comment:
and here please

Reply:
Odd. I thought `Test_normal_ctrl_square_bracket_right` should cross this code but that test passes even without this block. Might be a difference between Neovim and Vim. I'll follow up on this

Reply:
Neovim behaves the same as this branch. Removed unneeded code in both locations.

Reply:
okay thanks.